prov/efa/src/rdm/efa_rdm_ep_utils.c

```diff
@@ -661,12 +663,31 @@ ssize_t efa_rdm_ep_trigger_handshake(struct efa_rdm_ep *ep, struct efa_rdm_peer
 	efa_rdm_tracepoint(trigger_handshake_begin, 0, 0, txe->msg_id,
 			   (size_t) txe->cq_entry.op_context, txe->total_len);
 
-	err = efa_rdm_ope_post_send(txe, EFA_RDM_EAGER_RTW_PKT);
+	pkt_entry = efa_rdm_pke_alloc(ep, ep->efa_tx_pkt_pool, EFA_RDM_PKE_FROM_EFA_TX_POOL);
```

Comment:
We even do not need an txe to trigger handshake. But this can be a separate change